src/org/cru/jenkins/lib/EnvironmentLoader.groovy

```diff
@@ -31,12 +31,13 @@ class EnvironmentLoader implements Serializable {
       checkoutIfNecessary(ecsConfigDir)
 
       steps.bash """\
-        source \${ECS_CONFIG}/bin/load_environment.sh;
-        load_environment;
-        ${script}
-        """.stripIndent()
-    }
+        # Source rbenv to setup ruby in the path
+        eval "\$(~/.rbenv/bin/rbenv init -)"
 
+        # Execute script in an assumed environment
+        \${ECS_CONFIG}/bin/jenkins-pipeline -v -e \${ENVIRONMENT} -n \${PROJECT_NAME} -- "${script}"
```

Comment:
since you created long-form version of the args, would you mind using them here, for better readability? (i.e. `--env` instead of `-e`)